Python/libraries/recognizers-date-time/recognizers_date_time/date_time/english/date_parser_config.py

```diff
@@ -175,6 +175,10 @@ def get_swift_day(self, source: str) -> int:
             swift = 2
         elif trimmed_text.endswith('day before yesterday'):
             swift = -2
+        elif trimmed_text.endswith('day after') or trimmed_text.endswith('day after tmr'):
```

Comment:
This swift calculation for "tmr" seems wrong.
BTW, why is Python different from the others platforms? Shouldn't all handle this case?